PrebidMobile/ConfigurationAndTargeting/Prebid.swift

```diff
@@ -183,16 +180,27 @@ public class Prebid: NSObject {
     /// - Parameters:
     ///   - gadMobileAdsObject: GADMobileAds object
     ///   - completion: returns initialization status and optional error
+    @available(*, deprecated, message: "If the GMA SDK version is higher than 10.7.0 you should use `initializeSDK(gadMobileAdsVersion:, _ completion:)` method in order to initialize Prebid SDK.")
```

Comment:
We shouldn't deprecate this method so far because publishers still can use previous GMA SDK versions, and we shouldn't push them to migrate (via warning). 

We should log the message in the `checkGMAVersion` method if the publisher updated the GMA SDK but still use this initializer.